uber/automated_emails.py

```diff
@@ -64,9 +64,9 @@ def send_all(cls, raise_errors=False):
                 all_sent = set(session.query(Email.model, Email.fk_id, Email.subject))
                 for model, lister in cls.queries.items():
                     for inst in lister(session):
+                        sleep(0.01)  # throttle CPU usage
                         for rem in cls.instances.values():
                             if isinstance(inst, rem.model) and (not rem.needs_approval or rem.subject in approved):
-                                sleep(0.01)  # throttle our CPU usage
```

Comment:
I accidentally told it to sleep 10 milliseconds between each email/instance combination instead of between each instance.  Since we have 96 automated emails, that's the difference between less than a 5 minute delay and a many hour delay.